app/components/frame.js

```diff
@@ -23,7 +23,7 @@ export default class FrameComponent extends Component {
 
     @action
     updatePixelColor(pixel) {
-        pixel.color = this.paletteColor;
+        pixel.color = this.selectedColor;
```

Comment:
I think it might be worth making this component a Glimmer component; which would make this line: 

```suggestion
        pixel.color = this.args.selectedColor;
```

Reply:
Good call!